src/test/java/com/google/sps/servlets/TestAuthServlet.java

```diff
@@ -0,0 +1,63 @@
+package com.google.sps.servlets;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class TestAuthServlet {
+  @Mock
+  UserService mockUserService;
+  @Mock
+  HttpServletRequest mockRequest;
+  @Mock
+  HttpServletResponse mockResponse;
+  @Rule
+  public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+  @Test
+  public void testUserLoggedInCreatesLogOutLink() throws IOException {
```

Comment:
Please use the naming convention explained in https://github.com/marcbury-g/step/blob/master/appengine-di-skeleton/src/test/java/com/example/step/TestMyServlet.java
E.g.,
`testAuthServlet_doGet_WithLoggedInUser_ReturnsLogoutLink`